### PR TITLE
feat: button group variant "flat" and vertical orientation

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-id-mobile/issues/2#issuecomment-878087072 |
| Dependencies | -- |
| Decisions | • Button variant "flat" and align support <br> • Button toggle variant, align and new directions for vertical orientation. Support for no toggle behaviour and instead call a callback function. <br> • Button toggle variant and vertical orientation. |
| Animated GIF | button-group-variant<br>Usage example: <br>image|

### CODE REVIEW

File: react/components/molecules/button-toggle/button-toggle.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ export class ButtonToggle extends mix(PureComponent).with(IdentifiableMixin) {
             color: PropTypes.string,
             colorSecondary: PropTypes.string,
             value: PropTypes.bool,
-            orientation: PropTypes.string,
```
Comment (gcandal) [inline react/components/molecules/button-toggle/button-toggle.js:19]: onPress?

Comment (BeeMargarida) [reply react/components/molecules/button-toggle/button-toggle.js:19]: I'm using a logic of callback, where it receives a callback and calls it on onPress

Comment (BeeMargarida) [reply react/components/molecules/button-toggle/button-toggle.js:19]: I'll change to onPress

**Review by 3rdvision — APPROVED** [review]

File: react/components/organisms/button-group/button-group.js [hunk 2]
```diff
@@ -55,42 +65,62 @@ export class ButtonGroup extends mix(PureComponent).with(IdentifiableMixin) {
...
                     ...item.buttonProps,
                     loading: item.loading || this.props.loading,
                     disabled: item.disabled || this.props.disabled
                 }}
-                orientation={this._orientation(index)}
+                variant={this.props.variant}
+                direction={this._direction(index)}
+                align={this.props.align}
+                toggle={this.props.toggle}
+                onPress={item.callback}
```
Comment (3rdvision) [inline react/components/organisms/button-group/button-group.js:107]:
```suggestion
                onPress={item.onPress}
```
maybe

**Review by joamag — APPROVED** [review]

**Pull request merged**